docs/README.md

```diff
@@ -68,14 +68,61 @@ macOS (OS X) development environment.
 # User documentation
 
 * [Getting started (How-To with examples)](usage.md)
+
+## C/C++ Analysis
 * [Analyzer User guide](analyzer/user_guide.md)
-* [Webserver User guide](web/user_guide.md)
-* [Requiring credentials to view analysis results](web/authentication.md)
-* [Overview about connecting multiple analysis run databases](web/products.md)
-* [Permission management](web/permissions.md)
-* [Usage of PostgreSQL database](web/postgresql_setup.md)
-* [How to deal with false positives](analyzer/false_positives.md)
-* [Store Cppcheck analyzer reports](/tools/report-converter/README.md#cppcheck)
+* [Avoiding or suppressing false positives](analyzer/false_positives.md)
+* [Checker and Static Analyzer configuration](analyzer/checker_and_analyzer_configuration.md)
+* [Checker documentation](web/checker_docs.md)
+* [GCC incompatibilities](analyzer/gcc_incompatibilities.md)
+
+## Web based report management
```

Comment:
For me this long list is not so readable. I would highly recommend to structure this list by using sublist:

* [Webserver User Guide](web/user_guide.md)
* [Command line and WEB UI Feature overview](feature_comparison.md)
* Security configuration
  * [Configuring Authentication](web/authentication.md)
  * [Configuring Authorization](web/permissions.md)
* Deployment
  * [Deploy server using docker](web/docker.md#deployment)
* Continuous Integration (CI)
  * [Setting up CI gating with Gerrit and Jenkins](jenkins_gerrit_integration.md)
  * [Setting up CI gating with Gitlab](gitlab_integration.md)
* Server configuration
  * [Configuring Server Logging](logging.md)
  * [Setting up multiple CodeChecker repositories in one server](web/products.md)
* Database configuration
  * [PostgreSQL database backend setup guide](web/postgresql_setup.md)
  * [CodeChecker server and database schema upgrade guide](web/db_schema_guide.md)

What do you think @gyorb @dkrupp ?